# Handling_Errors_Throws
Use error handling with throws

A good Example for error handling with throws
